tests/cypress/e2e/helpers.ts

```diff
@@ -10,6 +10,11 @@ export function loginGrafana(grafanaUsername: string, passwordGrafana: string) {
   cy.get('[aria-label="Login button"]').click();
 }
 
+export function addNewPanel() {
```

Comment:
just a nitpick, not sure if it's worth the effort: you don't just add a new panel, but you also create a new dashboard, that contains the new panel. not sure how you could express this in the method name, or if it's really important here.

Reply:
Having a more verbose name didn't look very nice. I added a comment in the method to be more specific about what it actually does.